.github/workflows/deploy.yml

```
@@ -5,7 +5,7 @@ name: MyST GitHub Pages Deploy
 
 on:
   # Uncomment the 'pull_request' line below to manually re-build Jupyter Book
-  # pull_request:
+  pull_request:
```

```suggestion
  # pull_request:
```
TODO comment out this line again before merging.